app/carbonapi/app.go

```diff
@@ -498,7 +497,7 @@ func initBackend(config cfg.API, logger *zap.Logger) (backend.Backend, error) {
 
 	// TODO (grzkv): Stop using a list, move to a single value in config
 	if len(config.Backends) == 0 {
-		return nil, errors.New("got empty list of backends from config")
+		return nil, fmt.Errorf("got empty list of backends from config")
```

Comment:
There are no arguments for string, so `errors.New` is sufficient.

Reply:
Done!